File: test/java/arraysandstrings/CombineTwoArraysTest.java
package java.arraysandstrings;

import org.junit.jupiter.api.Test;

import static java.arraysandstrings.CombineTwoArrays.combine;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class CombineTwoArraysTest {

    @Test
    void combineTest() {
        assertArrayEquals(new int[] {1, 2, 3, 4, 5},
                combine(new int[]{3, 5}, new int[] {1, 2, 4}));
    }
}

File: src/test/kotlin/arraysandstrings/CombineTwoArraysTest.kt
package arraysandstrings

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class CombineTwoArraysTest {
    @Test
    fun combineTest() {
        Assertions.assertArrayEquals(
            intArrayOf(1, 2, 3, 4, 5),
            CombineTwoArrays.Companion.combine(intArrayOf(3, 5), intArrayOf(1, 2, 4))
        )
    }
}